### PR TITLE
Kim edit: acm-lightning-components.adoc

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
Kim edit: acm-lightning-components.adoc

### CODE REVIEW

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 1]
```diff
@@ -0,0 +1,240 @@
+= ACM Lightning Components
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:1]: It seems odd to me to have this Reference topic at the same level as customization (of which it is a part), previewing, and advanced styling.
But this is part of the larger conversation for later!  :-)

Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc:1]: How about the Description of ACM Lightning Design Components?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+To design your API Community Manager portal, open the Builder and add ACM Lightning Components as described in xref:customize.adoc[Customize the Look and Feel of Your API Community].
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:7]: This is a little misleading because there are other ways to design the...portal? or community? 
The previous topic states "For detailed component information, refer to ACM Lightning Components and the Salesforce Community Cloud Community Components documentation," and then goes on to describe other design components. It's a little unclear to me, as a complete newcoming, to understand how these all fit together. I wonder if we could include a conceptual diagram that details the design architecture....

Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc:7]: ...your API Community Manager portal,...
   > "...your API Community Manager ACM portal,..."    [Also, do we need to define what the "portal" is vs. a home page?]

Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc:7]: Consider omitting "as described in xref:customize.adoc[Customize the Look and Feel of Your API Community]." Use See Also link instead, if really necessary.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+
+This list explains each of the ACM Lightning Components, the functionality they add to the portal, and their configuration options.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:9]: GLOBALLY try to avoid self-referring text: "This list explains..."
Replace with something like "You should understand your options for configuring each component, as well as its role in the portal.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+== API Access Requester
+
+The API access requester button component enables members to request access to an API for an application.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:13]: button component > button

members > community members

Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc:13]: enables community members to either create a new application or request access to an API for an existing application.

Is this correct? If so, use this combo and omit the following sentence:

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+
+Members can request access for an existing client application and API instance, or request to create a new client application by specifying the application name and description and choosing an API instance.
+
+* *API*: Choose the API.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:17]: Since this is a Reference and not a Task, suggest rewording each entry to reflect that:

o API
   Provides a list of APIs from which you can choose
o Button Border Radius
   Enables you to set the radius of the button border....
o Button Align
    Specifies left, right, or center alignment....

GLOBAL in each of the following lists. I will stop marking this from here on.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+To design your API Community Manager portal, open the Builder and add ACM Lightning Components as described in xref:customize.adoc[Customize the Look and Feel of Your API Community].
+
+This list explains each of the ACM Lightning Components, the functionality they add to the portal, and their configuration options.
+
+== API Access Requester
+
+The API access requester button component enables members to request access to an API for an application.
+
+Members can request access for an existing client application and API instance, or request to create a new client application by specifying the application name and description and choosing an API instance.
+
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:16]: Per above comment, omit previous sentence, but introduce list with something like "You can specify values for the following parameters of the API Access Requester:"

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+* *Button Align*: Set button alignment to left, right, or center.
+* *Show Modal Title Line*: Show the title line in the access request modal.
+// check description of Show Modal Title Line
+
+== API Card
+
+The API card component shows information about an API, including icon, name, and description.
+
+* *API*: Choose the API to show.
+* *Override URL*: When the user clicks the *API Details* button on the card, a URL opens. If this optional field is set, the URL in the field overrides the default URL. This could be used to direct users to a documentation or marketing page about the API, which would also have a link to the API's main page.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:28]: ditto re. description vs. instruction wording

Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc:28]: The amount of text here is not parallel to peer list items. How about something like...:

If set, overrides the default URL to the main page of the API in favor of linking to other information that you want the user to see first, such as documentation or marketing collateral.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+* *Show Buttons*: Show the *API Details* button. If the button is not shown, users can click anywhere on the card to show the API details.
+* *Card Height*: Set the card height.
+* *Card Margin*: Set the amount of margin space around the card.
+* *Card Border*: Set the thickness of the card borders.
+* *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
+* *Card Shadow*: Set the shadow beneath the card.
+
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:35]: Show > Shows
Set > Sets
(function vs. user step)

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+== API Carousel
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:36]: ditto previous applicable GLOBAL suggestions

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+
+The API carousel component shows a horizontally scrollable set of API cards.
+
+* *Show Buttons*: Show the *API Details* button on each card. If the button is not shown, users can click anywhere on the card to show the API details.
+* *Max Cards Display*: Set the maximum number of cards to show.
+* *Rows*: Set the number of rows of cards in the carousel.
+* *Show Arrows*: Show arrows on the right and left sides of the carousel. Clicking the arrows scrolls through the cards.
+* *Arrow Size*: Set the size of the arrows on the right and left sides.
+* *Show Dots*: Show dots below the cards. Clicking the dots scrolls through the cards.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:45]: What is the difference between using the arrows to scroll and using the dots? Arrows seems intuitive to me; not sure I "get" what the dots imply, navigation-wise....

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 11]
```diff
@@ -0,0 +1,240 @@
...
+* *Card Height*: Set the card height.
+* *Card Margin*: Set the amount of margin space around each card.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:47]: Omit "amount of."

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 12]
```diff
@@ -0,0 +1,240 @@
...
+* *Card Border*: Set the thickness of the card borders.
+* *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
+* *Card Shadow*: Set the shadow beneath each card.
+* *Message If Empty*: Show this message if the carousel has no APIs. Default is "There are no APIs available right now."
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:51]: Just a note to dev: it would be better if the default text were "No APIs are available."

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 13]
```diff
@@ -0,0 +1,240 @@
...
+* *Filter Condition (Advanced)*: Filter APIs using package names, field names such as "API name" or "API description", and comparison of strings to those fields.
+// need better info on using Filter condition
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:53]: Yep. I don't get the last item, and it is not parallel to the previous two.
Consider omitting "such as API name or API description" : Filters APIs by using package names, field names, or [field strings?]"

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 14]
```diff
@@ -0,0 +1,240 @@
...
+* *Show Buttons*: Show the *API Details* button. If the button is not shown, users can click anywhere on the card to show the API details.
+* *Card Height*: Set the card height.
+* *Card Margin*: Set the amount of margin space around the card.
+* *Card Border*: Set the thickness of the card borders.
+* *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
+* *Card Shadow*: Set the shadow beneath the card.
+
+== API Carousel
+
+The API carousel component shows a horizontally scrollable set of API cards.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:38]: What about an adapted and labeled illustration of one of those old slide carousels? Not sure international audiences will get it.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 15]
```diff
@@ -0,0 +1,240 @@
...
+
+* *Show Buttons*: Show the *API Details* button on each card. If the button is not shown, users can click anywhere on the card to show the API details.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:40]: If the button is not shown....   Consider rewording to something like "Regardless of whether the button is shown, users can click...."

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 16]
```diff
@@ -0,0 +1,240 @@
...
+* *Max Cards Display*: Set the maximum number of cards to show.
+* *Rows*: Set the number of rows of cards in the carousel.
+* *Show Arrows*: Show arrows on the right and left sides of the carousel. Clicking the arrows scrolls through the cards.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:43]: Should we clarify that the right arrow scrolls right and the left, left...or is that too obvious?

"....scrolls through the cards in the direction of the arrow clicked."?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 17]
```diff
@@ -0,0 +1,240 @@
...
+* *Card Border*: Set the thickness of the card borders.
+* *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
+* *Card Shadow*: Set the shadow beneath each card.
+* *Message If Empty*: Show this message if the carousel has no APIs. Default is "There are no APIs available right now."
+* *Filter Condition (Advanced)*: Filter APIs using package names, field names such as "API name" or "API description", and comparison of strings to those fields.
+// need better info on using Filter condition
+
+== API Console
+
+The interactive API console component shows information about your APIs and their methods, and allows users to test the methods by sending requests.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:57]: allows > enables

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 18]
```diff
@@ -0,0 +1,240 @@
...
+
+* *API*: Choose the API to display.
+* *Show Documentation Pages*: Show the documentation pages for the API.
+* *Link Hover Color*: Set the color of links when the cursor hovers over them.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:61]: "Sets the color for the hover links."?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 19]
```diff
@@ -0,0 +1,240 @@
...
+* *Button Text Color (Optional)*: Set the color of text on buttons.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:62]: Sets the color of button text?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 20]
```diff
@@ -0,0 +1,240 @@
...
+* *Button Hover Text Color (Optional)*: Set the color of text on buttons when the cursor hovers over them. Default is the button text color.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:63]: of button hover text

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 21]
```diff
@@ -0,0 +1,240 @@
...
+* *Headline Font Family (Optional)*: Set the font family of the API console headline.
+* *Error Color (Optional)*: Set the color of error messages.
+* *Method Label Get Background Color*: Set the background color of the labels of *GET* methods.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:66]: Consider GLOBALLY changing "the labels of xyz" to "xyz labels."

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 22]
```diff
@@ -0,0 +1,240 @@
...
+* *Method Label Options Background Color*: Set the background color of the labels of *OPTIONS* methods.
+* *Method Label Options Color*: Set the color of the labels of *OPTIONS* methods.
+* *Method Label Head Background Color*: Set the background color of the labels of *HEAD* methods.
+* *Method Label Head Color*: Set the color of the labels of *HEAD* methods.
+
+// Check descriptions: *Headline font family (optional)*
+
+== API Console Documentation
+
+The API console documentation component shows an API's documentation in the API console.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:85]: ...shows documentation about APIs.

I wish we could add value by saying something that does not so directly repeat the heading....

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 23]
```diff
@@ -0,0 +1,240 @@
...
+
+* *API*: Choose the API to display.
+* *Show Documentation Pages*: Show the documentation pages for the API.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:88]: Show the API documentation pages.

GLOBALLY reevaluate the "for the" and the "of the" wordings and see if you can reorder them in the same way as suggested here and above. I'll stop marking this.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 24]
```diff
@@ -0,0 +1,240 @@
...
+
+The API documentation viewer component shows an API's documentation. The documentation may have multiple pages, and each page may have multiple sections.
+
+* *API*: Choose the API.
+* *Section (Optional)*: Set the section to open. If the *Selected page (optional)* setting is empty, the *Section (optional)* setting is ignored.
+* *Selected Page (Optional)*: Set the page to open.
+
+== API Header
+
+The API header component displays the API Header including icon, name, and description.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:172]: Header > header, including

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 25]
```diff
@@ -0,0 +1,240 @@
...
+
+* *API*: Choose the API to show.
+
+== Application Analytics Graph
+
+The application analytics graph component displays statistics about the use of an application.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:178]: application, such as <list a couple here>.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 26]
```diff
@@ -0,0 +1,240 @@
...
+
+== Application Details
+
+The application details component displays details about an application.
+
+// check component description
+
+* *Application*: Choose the API to show.
+
+== Application Listing
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:190]: The values below sound as if they apply only to the carousel; is that correct?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 27]
```diff
@@ -0,0 +1,240 @@
...
+* *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
+* *Card Shadow*: Set the shadow beneath the card.
+* *Card Width*: Set the card width.
+* *Card Height*: Set the card height.
+
+// check *New Application URL* description
+
+== Login Form
+
+The login form component allows members to log in to the portal.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:206]: allows > enables
members > community members

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 28]
```diff
@@ -0,0 +1,240 @@
...
+
+* *Start URL*: Set the URL members see after completing login. Default is the portal home page.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:208]: Sets the URL that members see after they log in. The default value is the portal home page.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 29]
```diff
@@ -0,0 +1,240 @@
...
+* *Username Label*: Set the label for the username field. Default is `Email`.
+* *Password Label*: Set the label for the password field. Default is `Password`.
+* *Login Button Label*: Set the label for the login button. Default is `Log in`.
+* *Forgot Password Label*: Set the label for the forgotten password link. Default is `Forgot your password?`.
+* *Forgot Password URL*: Set the URL for the forgotten password link. Default is `/ForgotPassword`.
+* *Self Register Label*: Set the label for the self registration link. Default is `Not a member?`.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:214]: Here and below, "self registration link" > "self-registration link"

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 30]
```diff
@@ -0,0 +1,240 @@
...
+* *Self Register URL*: Set the URL for the self registration link. Default is `/SelfRegister`.
+
+// check *Start URL* description
+
+== Search Result List
+
+The search result list component shows a list of search results. It has no configuration settings.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:221]: Should we add the second sentence to the others that have no config settings?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 31]
```diff
@@ -0,0 +1,240 @@
...
+
+== Self Register
+
+The self register component allows users to register for portal accounts.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:225]: allows > enables

**Review by kbostrom — APPROVED** [review]
Mismatch between "user verbs" and reference text

**Comment by wwarner-salesforce** [comment]
PR to address these changes: https://github.com/mulesoft/docs-api-community-manager/pull/95

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:1]: Salesforce uses the phrase "Lightning Component" as a proper noun, so I've changed the title of this page to "Description of ACM Lightning Components".

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:7]: Good point-- removed link and updated text to show the portal has multiple pages.

Portal vs. Community: If the ACM Portal is like a bumper car ride at a fair, with multiple fairgoers (consumer developers) interacting and using the machinery, the Salesforce Community is hidden away within the machinery that makes it work, like a diesel generator powering everything in the ride. I like the idea of adding more architectural diagrams, and this is on our current list for the ACM docs.

Adding and configuring ACM Lightning Components in the Builder is the only way I know to design the pages of an ACM portal.

The portal can be configured with both the default Salesforce Lightning Components that are available for all Salesforce Communities, and the ACM Lightning Components that are only available in ACM. Added this info.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:9]: Good point. Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:13]: Updated to this:

The **API Access Requester** component enables community members to request access to an API for either a new or existing application.

Community members can either request access for an existing client application and API instance, or request to create a new client application with access to the API by specifying the application name and description and choosing an API instance.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 5 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:17]: Good point! Fixed globally.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 6 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:16]: Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 7 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:28]: Much better! Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 8 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:35]: Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 9 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:36]: Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 10 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:45]: Good point. If there are four pages of cards, there will be four dots. Users can scroll to the next and previous pages with the arrows, or jump immediately to any page with the dots. Updated text to "Shows dots below the cards. Each dot links to one of the pages of cards in the carousel."

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 11 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:47]: Good point! Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 12 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:51]: Good point. I'll suggest this change.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 13 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:53]: True, "Filter Condition" is unlike any of the other options. It allows the user building the page to input a Salesforce Object Query Language (SOQL) search, and the carousel displays only the APIs that match the search.

MuleSoft and Salesforce employees have been asking for more info about using filter conditions in the ACM chat rooms, and Jesus has made plans to add a section on that, so soon we'll be able to refer to that section from here.

For now I've updated the text to "Limits the APIs shown in the carousel to those matching a standard Salesforce Object Query Language (SOQL) search. SOQL searches use package names, field names such as "API name" or "API description," and comparison of strings to the values of those fields."

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 14 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:38]: I like this idea! I hadn't even thought about the carousels of slides in a projector, and was only thinking of circus carousels. I've added this to the list of ideas for future improvements.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 15 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:40]: Updated to "Users can see API details by clicking the button if it is present or by clicking anywhere on the card if it is not."

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 16 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:43]: I think this helps clarity, so I've added it.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 17 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:57]: Good point! Changed this and the other instances of "allow" in the file to "enable."

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 18 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:61]: Sounds good. Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 19 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:62]: Much better! Fixed all instances of this in the file.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 20 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:63]: Fixed all instances of this in the file.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 21 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:66]: Agreed. Made this change globally.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 22 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:85]: Updated to "The **API Console Documentation** component shows the documentation from Anypoint Exchange about an API."

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 23 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:88]: That's a good point. Reordered those as suggested everywhere.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 24 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:172]: Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 25 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:178]: Added example statistics.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 26 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:190]: Each application is on a card, but I'm not sure if the cards are grouped in a carousel similar to the API carousel or just a grid that is made long enough to show all applications.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 27 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:206]: Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 28 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:208]: Much better! Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 29 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:214]: Fixed in all instances.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 30 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:221]: Yes, it should be added to all components that have no config settings. However, the Search Result List is the only component with no config settings.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 31 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:225]: Fixed.

**Comment by wwarner-salesforce** [comment]
Resolved in https://github.com/mulesoft/docs-api-community-manager/pull/95 .
